src/annotator/guest.js

```diff
@@ -379,6 +391,8 @@ export default class Guest {
   }
 
   destroy() {
+    this._annotations.clear();
+    this._deferredDetach.clear();
```

Comment:
Is there a need to clear these fields manually? There aren't any event handlers or other resources here which the GC can't reclaim automatically.